modules/measure/src/main/java/com/opengamma/strata/measure/StandardMeasures.java

```diff
@@ -34,6 +34,8 @@ final class StandardMeasures {
   public static final Measure CASH_FLOWS = ImmutableMeasure.of("CashFlows");
   // currency exposure, with no currency conversion
   public static final Measure CURRENCY_EXPOSURE = ImmutableMeasure.of("CurrencyExposure", false);
+  // delta
+  public static final Measure DELTA = ImmutableMeasure.of("Delta");
```

Comment:
Given that there are various measures people call delta, would OPTION_DELTA be a better name?